gcs/smart_entry.hh

```diff
@@ -0,0 +1,130 @@
+#ifndef GLASGOW_CONSTRAINT_SOLVER_GUARD_GCS_SMART_ENTRY_HH
+#define GLASGOW_CONSTRAINT_SOLVER_GUARD_GCS_SMART_ENTRY_HH
+
+#include <variant>
+#include <vector>
+#include <gcs/variable_id.hh>
+#include <gcs/integer.hh>
+#include <algorithm> //std::sort
```

Comment:
Blank line before

Reply:
Fixed in c62e8914eb9e01df2a147f01713ff505d7e905e0